src/main/java/com/cloudbees/jenkins/support/config/SupportAutomatedBundleConfiguration.java

```diff
@@ -179,13 +175,22 @@ public static List<Component> getApplicableComponents() {
     @POST
     public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
         Jenkins.get().checkPermission(Jenkins.ADMINISTER);
-        if (json.getBoolean("enabled") && !json.has("components")) {
+
+        if(isEnforcedDisabled()) {
+            return true;
```

Comment:
This is going to suppress unrelated config like https://github.com/jenkinsci/support-core-plugin/blob/210c61fc96090e780143d0e55097373fe51be16f/src/main/java/com/cloudbees/jenkins/support/config/SupportAutomatedBundleConfiguration.java#L190?

Reply:
It leaves the config untouched actually. I tested this.

* If you set `-Dcom.cloudbees.jenkins.support.SupportPlugin.AUTO_BUNDLE_PERIOD_HOURS=0` and then go to the config page and save, it leaves the entire configuration as is (including `com.cloudbees.jenkins.support.config.SupportAutomatedBundleConfiguration.xml`). So that if you then remove the system property, you are back to your original configuration that you had.

* If you set `-Dcom.cloudbees.jenkins.support.SupportPlugin.AUTO_BUNDLE_PERIOD_HOURS=2` and then go to the config page and save, it leaves only the `period` configuration as is (including `com.cloudbees.jenkins.support.config.SupportAutomatedBundleConfiguration.xml`). If you later remove the system property, the generation is still enabled and you are back to your original `period` value.

Does that makes sense ?

Reply:
Whatever you say—I am not familiar with the details of this form.